### PR TITLE
Allow specifying environment variables for runner in options

### LINKED ISSUE #64: Unable to supply environment variables in `runner` argument.
Reported by @dreyks [here](https://github.com/usabilityhub/rails-erb-loader/issues/63#issuecomment-381112052).

>
> > `options: { runner: 'DISABLE_SPRING=1 bin/rails runner' }`
>
> I tried this for other ENV var that i needed and this does not work. parseRunner splits by spaces so the ENV var becomes a file that spawn tries to call

Rather than improving the support for `runner` string, I think something like this would be most appropriate for the internal usage of [`spawn`](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options).

```js
options: {
  runner: {
    path: 'bin/rails',
    args: ['runner'],
    env: {
      ...process.env,
      DISABLE_SPRING: 1,
    }
  }
}
```

NOTE: When supplying `options.env` to `spawn` it will override the default of passing `process.env`, so this must be explicitly provided by config. This additionally provides the option of passing `{}` to prevent this default behaviour.

### PR DESCRIPTION
**Pull request by PikachuEXE**
Fixes #64

### CODE REVIEW

**Comment by PikachuEXE** [comment]
I noticed there is no CI for this project so I run all the "scripts" in `package.json` and they all pass

File: index.js [hunk 1]
```diff
@@ -70,14 +70,19 @@ function parseDependencies (source, root) {
  * output transformed source.
  */
 function transformSource (runner, config, source, map, callback) {
+  var subprocessEnv = {
+    stdio: ['pipe', 'pipe', process.stderr],
+    env: defaults({}, config.env || {}, process.env)
```
Comment (rhys-vdw) [inline index.js]: So just `env: config.env || process.env` here.

**Comment by rhys-vdw** [comment]
Thanks @PikachuEXE. I would prefer if it were possible for the consumer to override the default of using the existing `process.env`.

This means you'd have to do:

```js
options: {
  env: { ...process.env, DISABLE_SPRING: 1 }
}
```

Unless you think that could be a problem? My reasoning is that it gives the user control.

File: index.js (hunk 1 above)
Comment (rhys-vdw) [reply index.js]: Oops, not that... just `config.env` here, and instead specify `process.env` in [here](https://github.com/usabilityhub/rails-erb-loader/blob/c2d980199eba59ceb7d8401194c4992329ec40a2/index.js#L191-L195) where default options are specified.

**Comment by rhys-vdw** [comment]
Merged and released @PikachuEXE! But I forgot to add you to the contributors array! I'll do so now.

You've been invited to collaborate on this project.

**Comment by PikachuEXE** [comment]
Thanks! Accept :)

**Pull request merged**